apps/builder/app/builder/builder.tsx

```diff
@@ -294,24 +297,31 @@ export const Builder = ({
   const [loadingState, setLoadingState] = useState(() => $loadingState.get());
 
   useEffect(() => {
+    if (!isDesignMode) {
+      return;
+    }
```

Comment:
why not $isDesignMode.subscribe inside effect?

Reply:
Attached only in design mode. Before was subscribe I spent 1h trying to get why and how it;s working, and pretty sure it didn't work well